.github/workflows/go.yml

```
@@ -20,7 +20,7 @@ jobs:
         with:
           go-version: ${{ env.GOLANG }}
       - name: Setup lint
-        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.47.3
+        run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.52.2
```

What you have should work, but my preference is to use `go install`. The documentation for `golangci-lint` recommends using curl, but I think go install makes sense in this context since we use go install to set up gonomics dependencies.
```suggestion
        run: go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.52.2
```